packages/models/src/eservice/eserviceEvents.ts

```diff
@@ -1,17 +1,17 @@
 import { match } from "ts-pattern";
 import {
   ClonedEServiceAddedV1,
-  EServiceUpdatedV1,
-  EServiceWithDescriptorsDeletedV1,
-  EServiceDocumentUpdatedV1,
+  EServiceAddedV1,
   EServiceDeletedV1,
-  EServiceDocumentAddedV1,
-  EServiceDocumentDeletedV1,
   EServiceDescriptorAddedV1,
   EServiceDescriptorUpdatedV1,
+  EServiceDocumentAddedV1,
+  EServiceDocumentDeletedV1,
+  EServiceDocumentUpdatedV1,
+  EServiceUpdatedV1,
+  EServiceWithDescriptorsDeletedV1,
   MovedAttributesFromEserviceToDescriptorsV1,
-  EServiceAddedV1,
-} from "./gen/v1/eservice/events.js";
+} from "../gen/v1/eservice/events.js";
 
 export function toBinaryData(event: EServiceEvent): Uint8Array {
```

Comment:
not strictly related to this PR but maybe we should change `toBinaryData` to`catalogEventToBinaryData`